libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -210,6 +211,9 @@ class BaseChatModel(BaseLanguageModel[BaseMessage], ABC):
     callback_manager: Optional[BaseCallbackManager] = Field(default=None, exclude=True)
     """[DEPRECATED] Callback manager to add to the run trace."""
 
+    rate_limiter: Optional[BaseRateLimiter] = None
```

Comment:
do we need to think about serialization?

Reply:
yes, great catch! I was checking this incorrectly with a non serializable model (and indirectly). Fixed now and set this to exclude=True.